packages/react-component-library/src/components/ClassificationBar/ClassificationBar.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { ComponentWithClass } from '../../common/ComponentWithClass'
+import { StyledClassificationBar } from './partials/StyledClassificationBar'
+
+export interface ClassificationProps extends ComponentWithClass {
+  isSecret?: boolean
+}
+
+export const ClassificationBar = ({
+  isSecret,
+  ...rest
+}: ClassificationProps) => {
+  const classificationText = isSecret ? 'Secret' : 'Official Sensitive'
+  return (
+    <StyledClassificationBar
+      data-testid="classification-bar"
```

Comment:
I wonder if it's possible to test this without a `data-testid`.

Test IDs are used extensively in this code base but it's actually a bad practice we want to get away from.

The problem is they couple our tests tightly to the implementation details of the code.

A good alternative is to use the more accessible `*ByRole` and `*ByText` RTL matchers.

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-querying-by-text

Reply:
Interesting, i'll come back to this when I actually write the tests but makes sense with the tight coupling. Just put that in as I was trying to match some of the other components.